rise-storage.html

```diff
@@ -282,6 +282,16 @@
      */
     _folderFilesRequested: 0,
 
+
+    /**
+     * Whether there is a file from a folder still being downloaded by Cache.
+     *
+     * @property _hasFolderFileDownloadingByCache
+     * @type boolean
+     * @default false
+     */
+    _hasFolderFileDownloadingByCache: false,
```

Comment:
@rodrigopavezi Can we change this name to `isFolderFileDownloadingByCache`? Cheers